init/init-lib.pl

```diff
@@ -2178,7 +2181,7 @@ sub list_systemd_services
 	while(@args < 100 && @units) {
 		push(@args, shift(@units));
 		}
-	my $out = &backquote_command("systemctl show ".join(" ", @args)." 2>/dev/null");
+	my $out = &backquote_command("systemctl show --property=Id,Description,UnitFileState,ActiveState,SubState,ExecStart,ExecStop,ExecReload,ExecMainPID,FragmentPath ".join(" ", @args)." 2>/dev/null");
```

Comment:
Are you sure that it's safe to request specific properties if units without them would be skipped?

Reply:
Yes, it's safe, as we need units to have the `Id` field at least. Also, there are no such units out of 300+ on Ubuntu which would be skipped with `--properties` code vs show all. And it should be much faster with `--properties`, as otherwise there is a lot of lines to iterate.

Reply:
Have you tested to see what happens if you request a property that some units don't have?

Reply:
Yes, nothing happens:

```
root@ubuntu22-pro:~# systemctl show --property=Id,Description,UnknownProp webmin.service
Id=webmin.service
Description=Webmin server daemon
```

.. only known properties will be returned, and others are ignored.

Reply:
Ok! Do I still need to merge this PR, or have all the fixes been submitted?

Reply:
Yes, you need to merge it. 😅